pkgs/development/python-modules/mautrix/default.nix

```diff
@@ -34,13 +37,15 @@ buildPythonPackage rec {
     hash = "sha256-g6y2u3ipSp5HoakHqd/ryPlyA+kR7zO6uY4AqfqbwiE=";
   };
 
-  propagatedBuildInputs = [
+  build-system = [ setuptools ];
+
+  dependencies = [
     aiohttp
     attrs
     yarl
-  ];
+  ] ++ lib.optionals withOlm optional-dependencies.encryption;
 
-  passthru.optional-dependencies = {
+  optional-dependencies = {
```

Comment:
Isn't this supposed to be under `passthru`?

Reply:
Sandro [told me to change it](https://github.com/NixOS/nixpkgs/pull/336901#discussion_r1731098403). I guess this way is new?

Reply:
Apparently yes, had to look around a bit before I found this pattern elsewhere, sorry!